### PR TITLE
Add Flockpanel Stats tab

### PR DESCRIPTION
**Pull request by FlameArrow57**
[FEATURE]

## About the PR 
This PR adds a Stats tab to the Flockmind's Flockpanel

image

## Why's this needed? 
Thought it would be cool

### CODE REVIEW

File: code/datums/flock/flock.dm [hunk 1]
```diff
@@ -169,6 +179,21 @@ var/flock_signal_unleashed = FALSE
 					// enemy no longer exists, let's do something about that
 					src.enemies -= name
 
+		if ("stats")
+			var/list/stats = list(
+				"Drones realized: " = src.drones_made,
+				"Bits formed: " = src.bits_made,
+				"Net deaths: " = src.deaths,
```
Comment (TobleroneSwordfish) [inline code/datums/flock/flock.dm:186]: What about this is net?

File: code/datums/flock/flock.dm [hunk 2]
```diff
@@ -169,6 +179,21 @@ var/flock_signal_unleashed = FALSE
...
+				"Total resources gained: " = src.net_resources_gained,
+				"Net partitions: " = src.partitions_made,
```
Comment (TobleroneSwordfish) [inline code/datums/flock/flock.dm:188]: ditto

File: code/datums/flock/flock.dm [hunk 3]
```diff
@@ -43,6 +43,15 @@ var/flock_signal_unleashed = FALSE
 	var/datum/tgui/flockpanel
 	var/ui_tab = "drones"
 
+	// stats stuff, if not listed above
+	var/drones_made = 0
+	var/bits_made = 0
+	var/deaths = 0
+	var/net_resources_gained = 0
```
Comment (TobleroneSwordfish) [inline code/datums/flock/flock.dm:50]: ditto

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
Might be nice to separate this into unit stats and structure stats as two side-by-side panels instead of one big list

File: code/datums/flock/flock.dm (hunk 1 above)
Comment (FlameArrow57) [reply code/datums/flock/flock.dm:186]: Just total drone + bit deaths

File: code/datums/flock/flock.dm (hunk 2 above)
Comment (FlameArrow57) [reply code/datums/flock/flock.dm:188]: Total number of partitions created

File: code/datums/flock/flock.dm (hunk 3 above)
Comment (FlameArrow57) [reply code/datums/flock/flock.dm:50]: Total accumulated resources

**Comment by FlameArrow57** [comment]
I don't know what structure stats would involve, other than maybe number of human caged, but that's a core structure so idk

File: code/datums/flock/flock.dm (hunk 1 above)
Comment (TobleroneSwordfish) [reply code/datums/flock/flock.dm:186]: then call it total and not net? that's not what net means.

**Pull request merged**